DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: rhino
 Title: A Framework for Enterprise Shiny Applications
-Version: 1.1.1
+Version: 1.1.1.9999
```

Comment:
Thanks for bringing this up. I considered using such a versioning scheme in the past, but I was hesitant for two reasons:
1. It does not adhere to [Semantic Versioning](https://semver.org/).
2. With our current configuration, it will appear at the top of our [documentation](https://appsilon.github.io/rhino/) once we merge.

On the other hand:
1. Versions like `1.1.1-dev.1` (adhering to semver) are not allowed in `DESCRIPTION`.
2. The `.9000` component is a kind of standard in R packages.
3. Clearly marking the dev version seems quite beneficial (whether for docs or installed package).
4. We shall be able to change our configuration to render the docs from the latest stable version, if we choose to do so. For now it should be fine to show the development version.

So yeah, let's start using this versioning scheme :grin: Let's go with `9000` though and increment it in each pull request, so different dev versions are clearly distinguishable.
```suggestion
Version: 1.1.1.9000
```